src/sentry/static/sentry/app/components/charts/tableChart.jsx

```diff
@@ -409,6 +406,10 @@ export const Cell = styled('div')`
   ${p => (p.justify === 'right' ? 'text-align: right' : '')};
 `;
 
+const Flex = styled('div')`
+  display: flex;
+`;
+
```

Comment:
How about just adding `display: flex` to the 3 styled components? same # of lines, and a little clearer I think